ansible-ipi-install/README.md

```diff
@@ -318,13 +318,47 @@ Sample playbook.yml
   - installer
 ~~~
 
+## Customizing the Node Filesystems
+If you need to modify files on the node filesystems, you can augment the "fake"
+roots for the masters and workers under the 
+`roles/installer/files/customize_filesystem/{master,worker}` directories. 
+Any files added here will be included in the ignition config files for each
+of the machine types, leading to permanent changes to the node filesystem.
+
+NOTE: Do not place any files directly in the "fake" root -- only in subdirectories.
+Files in the root will cause the ignition process to fail. (There is a task in the 
+playbook to cleanup the `.gitkeep` file in the root, if it is left in place.)
+
+This will utilize the Ignition 
+[filetranspiler tool](https://github.com/ashcrow/filetranspiler/blob/master/filetranspile), 
+which you can read about for more information on how to use the "fake" root directories.
+
+An example of using this customization is to disable a network interface that
```

Comment:
Will this example filesystem change also require MCO manifests (to be include as directed in `roles/installer/files/manifests`) for post-first-boot consistency?  When we ran into this problem using UPI last year (wanting to disable an external interface), we found that we had to change the ignition and add manifests for the MCO.  Otherwise the MCO would take issue with the filesystem being in an unexpected state and the deployment would fail.

Reply:
I haven't experienced this, but that's not to say it won't happen. I can confirm that this process, as defined here, will allow me to get master nodes deployed with the Ansible-IPI automation, and I can do this same ignition config trick with a manual UPI method and have it work for all master and worker nodes. So if you are saying that the actual deployment will not complete without the manifest changes, I don't believe that affects us.

However, our clusters have so far been short-lived, so if this is something that could affect the MCO post-install, then it is worth exploring further.

Reply:
The error described above was definitely something that appeared later in the deployment process.  I'll give these changes a go after this PR is merged and see if I encounter the same issue.  If I recall correctly, the MCO errors-out and thus the cluster operators never reach 100% availability.

Reply:
@abays - before I merge can you attempt this PR? From my view its merge ready but if we can see if you run into your issue before we merge makes my life easier :D 

```
git checkout -b dustinblack-shared_labs master
git pull https://github.com/dustinblack/baremetal-deploy.git shared_labs
```

Reply:
Sure, I will give it a try later today (I'm out at the moment).  My concern above was mainly with the example given as opposed to the general mechanism itself, though.  I otherwise agree that the PR looks good to go. :-)

Reply:
Well, good news: I'm not seeing the issue I was expecting.  Perhaps the behavior of the MCO has been changed.  Anyhow, it's working for me!